tests/vppcallmock/vpp_ctx_mock.go

```diff
@@ -26,7 +26,7 @@ import (
 //TestCtx is helping structure for unit testing. It wraps VppAdapter which is used instead of real VPP
 type TestCtx struct {
 	MockVpp     *mock.VppAdapter
-	conn        *core.Connection
+	Connection  *core.Connection
```

Comment:
Maybe don't export this since it wasn't intended to be used outside this package.

Reply:
It can be done without it using mocked context to set configurator's vpp channels directly (and skip init). 
The connection was the easy access to govppmux.API and let the configurator to initialize itself .